components/org.apache.stratos.manager/src/main/java/org/apache/stratos/manager/persistence/PersistenceManager.java

```diff
@@ -0,0 +1,112 @@
+package org.apache.stratos.manager.persistence;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.stratos.manager.exception.ApplicationSignUpException;
+import org.apache.stratos.manager.model.ApplicationSignUpEntity;
+import org.apache.stratos.manager.model.ApplicationSignUpEntityPK;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+
+import javax.persistence.Persistence;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by aarthy on 7/6/15.
+ */
```

Comment:
@JaneAarthy we don't put created by comment. We only put the Apache License header for the classes.